contracts/PrizeTierHistoryV2.sol

```diff
@@ -0,0 +1,110 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity 0.8.6;
+import "@pooltogether/owner-manager-contracts/contracts/Manageable.sol";
+import "./abstract/IdBinarySearch.sol";
+
+contract PrizeTierHistoryV2 is IdBinarySearch, Manageable {
```

Comment:
If the *interface* hasn't changed, let's just update the existing PrizeTierHistory contract

Reply:
Inherited.